administrator/components/com_associations/controllers/association.php

```diff
@@ -29,9 +31,18 @@ class AssociationsControllerAssociation extends JControllerForm
 	 */
 	public function edit($key = null, $urlVar = null)
 	{
-		JSession::checkToken() or jexit(JText::_('JINVALID_TOKEN'));
+		//JSession::checkToken() or jexit(JText::_('JINVALID_TOKEN'));
 		$cp = AssociationsHelper::getComponentProperties($this->input->get('component', '', 'string'));
 
+		$table = clone $cp->table;
+		$table->load($this->input->get('id', null, 'int'));
```

Comment:
It gives you an jtable instance of the table type without any data, so this is quite valid to useful sometimes.